lib/pharos/phases/configure_client.rb

```diff
@@ -47,11 +54,11 @@ def build_config(configmap)
         Pharos::Config.new(data)
       end
 
-      # @return [Kubeclient::Resource]
+      # @return [Pharos::Kube::Resource, nil]
```

Comment:
nil -> NilClass

Reply:
The first example at https://www.rubydoc.info/gems/yard/file/docs/Tags.md#Types_Specifier_List actually uses `nil`:

> ```
> # @return [String, Array<String>, nil] the object or objects to
> #   find in the database. Can be nil.
> ```